### PR TITLE
Integrated GDrive - save salon along with video file

### LINKED ISSUE #26: Integrate upload from Google Drive into Salon Create page
Screen Shot 2021-08-19 at 20 20 30

### PR DESCRIPTION
**Pull request by dragonfly91**

### CODE REVIEW

File: salon_creator/lib/screens/gdrive_picker_screen.dart [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  Widget _buildFileItem(CloudFile file) {
+    return ListTile(
+      subtitle: OutlinedButton(
+        child: file.thumbnailUrl != null
+            ? Image.network(
+                file.thumbnailUrl,
+                errorBuilder: (context, exception, stackTrace) => Container(),
+              )
+            : Text(file.name),
+        onPressed: () async {
```
Comment (mazemaze) [inline salon_creator/lib/screens/gdrive_picker_screen.dart:111]:
```suggestion
        onPressed: () async {
```
onPressed is too long.

**Review by mazemaze — CHANGES_REQUESTED** [review]
There's only one thing to fix, but otherwise Looks good to me

File: salon_creator/lib/screens/gdrive_picker_screen.dart (hunk 1 above)
Comment (dragonfly91) [reply salon_creator/lib/screens/gdrive_picker_screen.dart:111]: modified, please check!

**Review by mazemaze — APPROVED** [review]
LGTM!

File: salon_creator/lib/main.dart [hunk 2]
```diff
@@ -1,9 +1,22 @@
+import 'dart:io';
+
 import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
 import 'package:salon_creator/app.dart';
 
+class MyHttpOverrides extends HttpOverrides {
```
Comment (Hiroki0613) [inline salon_creator/lib/main.dart:7]: It is better to put a comment. "Only use at develop"

File: salon_creator/lib/models/media_item.dart [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class MediaItem {}
```
Comment (Hiroki0613) [inline salon_creator/lib/models/media_item.dart:1]: It is better to add a comment of structure.

File: salon_creator/lib/screens/gdrive_transfer_screen.dart [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    var fileId = widget.cloudFile.id;
+    
+    var transferDetails = TransferDetails(
+      id: fileId,
+      fileId: fileId,
+      status: TransferStatus.Successful,
+      thumbnailUrl: await storageHandler.getCloudFileThumbnailUrl(fileId),
+    );
+
+    setState(() {
```
Comment (Hiroki0613) [inline salon_creator/lib/screens/gdrive_transfer_screen.dart:85]: There are two setState for operationInProgress but I think first 
・operationInProgress = true 
is countered by second
・operationInProgress = false

**Review by Hiroki0613 — COMMENTED** [review]
Minor content. About adding comments.

**Pull request merged**